core/contracts/test/MockOracle.sol

```diff
@@ -41,12 +41,15 @@ contract MockOracle is OracleInterface, Testable {
     mapping(bytes32 => mapping(uint => QueryIndex)) private queryIndices;
     QueryPoint[] private requestedPrices;
 
+    // TODO: Is this solhint command required anymore?
```

Comment:
Nope, feel free to remove them when you see them.

Reply:
Oops forgot to resolve this TODO myself, done